server/publicapi/demo_data/data.json

```diff
@@ -0,0 +1,483 @@
+{
+    "_id": "tag:demodata.org,0001:ninjs_XYZ123",
+    "uri": "http://new.demodata.org/archive/20150316pic1234",
```

Comment:
Good catch, I'll update.

(we'll probably need to set this field on the fly from the `self` link that Eve generates)